shoop/core/order_creator/_source.py

```diff
@@ -207,6 +210,47 @@ def status(self):
     def status(self, status):
         self.status_id = (status.id if status else None)
 
+    @property
+    def codes(self):
```

Comment:
Missing docstring.

I do believe this used to be a set; why change it to a list?